chart/kubeapps/templates/tests/test-chartsvc.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ .Release.Name }}-chartsvc-test"
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+  - name: {{ .Release.Name }}-chartsvc-test
+    image: appropriate/curl
```

Comment:
Can we use an image that we have more control over here, such as bitnami/minideb? It's a little bigger, but hopefully not that big of a deal for running `helm test`.